lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -179,14 +179,17 @@ class UnusedCSSRules extends Audit {
       return UnusedCSSRules.mapSheetToResult(indexedSheets[sheetId], pageUrl);
     }).filter(sheet => sheet && sheet.wastedBytes > 1024);
 
+
+    const headings = [
+      {key: 'url', itemType: 'url', text: 'URL'},
```

Comment:
we can override this per row correct? it'd be nice to finally have a nice solution for displaying the inline code preview in the url column